.github/workflows/unittests.yml

```diff
@@ -61,13 +61,19 @@ jobs:
         run: |
           curl -L https://github.com/bazelbuild/bazelisk/releases/download/v1.3.0/bazelisk-darwin-amd64 > bazelisk
           chmod +x bazelisk
+      - name: Install GNU find
+        run: |
+          HOMEBREW_NO_AUTO_UPDATE=1 brew install findutils
+          ln -sf /usr/local/bin/gfind /usr/local/bin/find
       - uses: actions/setup-python@v1
         with:
           python-version: 3.7
       - name: Configure Bazel
         run: ./configure.sh
       - name: Install pip dependencies
         run: pip install tensorflow==2.1.0 larq_zoo==0.5.0 pytest tensorflow_datasets==1.3.2 --no-cache
+      - name: Run FileCheck tests
+        run: ./bazelisk test larq_compute_engine/mlir/tests:all --test_output=all --distinct_host_configuration=false
```

Comment:
Depending on the runtime we might want to run this in parallel in a new job.

Reply:
This now runs in parallel, which should give us faster CI failures when we mess things up.